lsp/src/Language/LSP/Server/Control.hs

```diff
@@ -187,23 +186,10 @@ ioLoop ioLogger logger clientIn serverDefinition vfs sendMsg = do
               go (parse parser remainder)
 
     parser = do
-      try contentType <|> (return ())
-      len <- contentLength
-      try contentType <|> (return ())
-      _ <- string _ONE_CRLF
-      Attoparsec.take len
-
-    contentLength = do
       _ <- string "Content-Length: "
       len <- decimal
-      _ <- string _ONE_CRLF
-      return len
-
-    contentType = do 
-      _ <- string "Content-Type: "
```

Comment:
This PR accidentally reverted #462 and therefore regressed #461 again: The `Content-Type` header is no longer parsed.

Can you please reapply the patch from #462?

Reply:
Argh, not sure how that happened. will fix

Reply:
https://github.com/haskell/lsp/pull/516